src/Surfnet/ServiceProviderDashboard/Domain/ValueObject/Issue.php

```diff
@@ -29,6 +29,7 @@ class Issue implements JsonSerializable
 
     const STATUS_CLOSED = 'CLOSED';
     const STATUS_RESOLVED = 'RESOLVED';
+    const STATUS_OPEN = 'To Do';
```

Comment:
I'm a tad confused by this: there's a 'OPEN' ticket status in the file below, but here the status_open constant is assigned the value To Do?

Reply:
@quartje provided an actual jira ticket dump, and turns out they use "To Do" for the open state. So, missed that on the DevIssueRepo.